src/metatrain/utils/io.py

```diff
@@ -99,15 +101,62 @@ def load_model(
         )
 
     if Path(path).suffix in [".yaml", ".yml"]:
-        raise ValueError(f"path '{path}' seems to be a YAML option file and no model")
+        raise ValueError(
+            f"path '{path}' seems to be a YAML option file and not a model"
+        )
 
-    if urlparse(str(path)).scheme:
+    # Download from HuggingFace with a private token
+    if kwargs.get("huggingface_api_token"):
+        try:
+            from huggingface_hub import hf_hub_download
+        except ImportError:
+            raise ImportError(
+                "To download a model from HuggingFace, please install the "
+                "`huggingface_hub` package with pip (`pip install "
+                "huggingface_hub`)."
+            )
+        path = str(path)
+        if not path.startswith("https://huggingface.co/"):
+            raise ValueError(
+                f"Invalid URL '{path}'. HuggingFace models should start with "
+                "'https://huggingface.co/'."
+            )
+        # get repo_id and filename
+        split_path = path.split("/")
+        repo_id = f"{split_path[3]}/{split_path[4]}"  # org/repo
+        filename = ""
```

Comment:
Instead of doing string manipulation, should we pass more structured information to this code? Copied from my comment on the overall PR:

maybe a better interface here would look like `mtt export --mode huggingface --hf-api-token=.... --hf-owner=who/what --hf-file=file-path` (very bad syntax, just for the idea. This could also be `huggingface://who/what/file-path` or `huggingface://token@who/what/file-path` or whatever).

I.e. instead of trying to guess the info we need, ask the use about it

Reply:
I don't think so because HuggingFace gives you the download link, so that's what most users will use

Reply:
Asking for the "split information" on the command line is just more work for the user